x/dsmr/node.go

```diff
@@ -171,7 +171,7 @@ func (n *Node[T]) NewChunk(
 	return chunk, n.storage.AddLocalChunkWithCert(chunk, &chunkCert)
 }
 
-func (n *Node[T]) NewBlock(parent Block, timestamp int64) (Block, error) {
+func (n *Node[T]) BuildBlock(parent Block, timestamp int64) (Block, error) {
```

Comment:
If we're going to name this `BuildBlock` we should also rename `NewChunk` to `BuildChunk` to be consistent

Reply:
Will do